cmd/root.go

```diff
@@ -17,11 +19,27 @@ import (
 
 var (
 	server, cfgFile string
+	Version         string = "dev"
 )
 
+func computeVersion() string {
```

Comment:
check out the link i sent before, i'll paste here again: https://www.digitalocean.com/community/tutorials/using-ldflags-to-set-version-information-for-go-applications

this whole function is not needed.